test/bats/akeyless.bats

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bats
+
+ load helpers
+
+ BATS_TESTS_DIR=test/bats/tests/akeyless
+ WAIT_TIME=120
+ SLEEP_TIME=1
+
+
+ export LABEL_VALUE=${LABEL_VALUE:-"test"}
+ export ANNOTATION_VALUE=${ANNOTATION_VALUE:-"app=test"}
+
+@test "install akeyless csi provider" {
+   # create the ns akeyless
+   kubectl create ns akeyless
+   # install the akeyless provider using the helm charts
+
+   helm repo add akeyless https://akeylesslabs.github.io/helm-charts
+   helm repo update
+   helm install akeyless akeyless/akeyless-csi-provider --namespace=akeyless --set csi.image.pullPolicy=Always
```

Comment:
Lets use `--create-namespace` flag here with helm install instead of creating namespace [here](https://github.com/kubernetes-sigs/secrets-store-csi-driver/pull/910/files#diff-882da5c47eff475fd29efc19d199866354b582b7e713fce8586fccb8911d5f05R15).

Reply:
Why are we setting image pull policy `Always`?

Reply:
fixed